robocop/checkers/naming.py

```diff
@@ -58,48 +61,197 @@ def visit_KeywordName(self, node):  # noqa
         self.check_if_char_in_node_name(node, 'KEYWORD_NAME')
 
 
-class CapitalizedNamesChecker(VisitorChecker):
-    """ Checker for capitalized keywords violation. """
+class KeywordNamingChecker(VisitorChecker):
+    """ Checker for keyword naming violations. """
     rules = {
         "0302": (
             "not-capitalized-keyword-name",
             "Keyword name should be capitalized",
             RuleSeverity.WARNING
+        ),
+        "0303": (
+            "keyword-name-is-reserved-word",
+            "'%s' is a reserved keyword%s",
```

Comment:
Lacking space before injected str?

Reply:
Last injected string can be empty - for 'while' and 'continue' reserved words (which don't have 'reserved_type' like for -> For loop, else -> Run keyword if etc).